stable_bot.py

```
@@ -58,135 +67,147 @@ def _transcribe_wav(path: str, language: str = 'ru-RU') -> str:
+async def memory(callback_query: CallbackQuery):
     if callback_query.data == 'remember':
-        await callback_query.answer('Правила: Ключ писать без пробелов, затем пробел и значение без пробелов',
-                                    show_alert=True)
+        await callback_query.message.answer('Пишите')
         keyboard_input[callback_query.from_user.id] = 'remember'
-        await callback_query.message.answer('Пример:\nцвет голубой'
-                                            '\nпокупки голубика_клубника_помидоры')
 
     elif callback_query.data == 'recall':
         await callback_query.answer('Вспоминаю...')
         asyncio.create_task(send_typing_indicator(callback_query.message.chat.id, callback_query.bot))
```

Ruff (0.14.8)

Store a reference to the return value of `asyncio.create_task`

(RUF006)